### PR TITLE
Catch and log exceptions occurring when decoding a delayed job's parameters

### PR DESCRIPTION
**Pull request by alexispeter**
fixes https://github.com/cargomedia/sk/issues/4547

### CODE REVIEW

File: library/CM/Jobdistribution/DelayedQueue.php [hunk 1]
```diff
@@ -53,9 +53,21 @@ public function queueOutstanding() {
...
-            $job = $this->_instantiateJob($row['className']);
+            $jobName = $row['className'];
+            $job = $this->_instantiateJob($jobName);
             if ($job) {
-                $job->queue(CM_Params::decode($row['params'], true));
+                $params = null;
+                $paramsEncoded = $row['params'];
+                try {
+                    $params = CM_Params::decode($paramsEncoded, true);
+                } catch (CM_Exception $ex) {
```
Comment (njam) [inline library/CM/Jobdistribution/DelayedQueue.php]: Shouldn't we catch *any* exceptions here? So that processing continues no matter what?

Comment (alexispeter) [inline library/CM/Jobdistribution/DelayedQueue.php]: True.

**Comment by njam** [comment]
@fvovan could you review this?

**Comment by fvovan** [comment]
lgtm

**Pull request merged**